api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ServiceRequestTranslatorImpl.java

```diff
@@ -109,7 +109,7 @@ public ServiceRequest toFhirResource(@Nonnull TestOrder order) {
 	
 	@Override
 	public TestOrder toOpenmrsType(@Nonnull ServiceRequest resource) {
-		throw new UnsupportedOperationException();
+		throw new UnsupportedOperationException("just checking why service request is not accepting patch operations!");
```

Comment:
I added this message just as I was stepping through to find out why the patch operation on this resource keeps returning a `failed to call access method: java.lang.UnsupportedOperationException`

Maybe you could have a better explanation @ibacher @abertnamanya @mogoodrich